### DISCUSSION TITLE
Orange PI ZERO 3 no boot

### DISCUSSION THREAD

**Discussion by brunofr29** [johang/sd-card-images#152] [Q&A] [answered]
hello, I try debian trixie on orange pi zero 3 [https://sd-card-images.johang.se/boards/orange_pi_zero3.html](url)
but it doen't run, it stop at starting kernel , see attached file.
[sd card image.txt](https://github.com/user-attachments/files/17530154/sd.card.image.txt)
can someone help me?
Thanks in advance
Bruno , french user of orangepi

**Comment by johang** [comment, marked as answer]
Try the the other images. Might be an issue with the Trixie kernel.

**Reply by brunofr29** [reply]
Dear Johang I tried Trixie Kernel but it's the same thing "starting Kernel" see attached file 
[debian.txt](https://github.com/user-attachments/files/17893071/debian.txt)
